src/odemis/driver/test/semcomedi_test.py

```diff
@@ -203,6 +205,12 @@ def compute_expected_duration(self):
         size = self.scanner.resolution.value
         return size[0] * size[1] * dwell + size[1] * settle
 
+    def test_ttl(self):
+        # Just check the VA are created and support changing the values
+        for v in (True, False, None):
+            self.scanner.external.value = v
+            self.scanner.blanker.value = v
```

Comment:
I am not sure how it is finally tested. If it wasn't a test case, you would raise errors or indicate that VA is not created.

Reply:
Yes, here it's a little hard to check the behaviour, as the test case cannot directly check the behaviour (ie reading the digital output). So it's just a matter of checking the VAs are there, and that they accept 3 values and that when setting it, nothing really bad happens. At least it runs the code.